src/main/resources/logback.xml

```diff
@@ -1,5 +1,6 @@
 <configuration>
   <property name="QUERYRESULT_LOG_FILE" value="logging/query-results" />
+  <springProperty name="ENCRYPT" source="app.resultLogEncryption.enabled"/>
```

Comment:
I chose the detour via spring application properties in order to set a default there